sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -234,9 +241,11 @@ export class SenderImpl implements Sender {
     }
 
     const scheduleMessageOperationPromise = async () => {
-      return this._context.managementClient!.scheduleMessages(
+      return this._context.managementClients[this._entityPath].scheduleMessages(
```

Comment:
Apparently this compiles but I'm surprised it's not complaining that the _managementClients[somevalue] could be null?

Reply:
With the accessor creating one if it does not exist, this wont be null. https://github.com/Azure/azure-sdk-for-js/pull/9710/commits/ed41c32649fd20ca340d73eb8cd032660dc06a15

Reply:
Yeah, that fixes it. I'm just curious why TS didn't complain when it was just an array.